release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -76,6 +76,12 @@ To install a CSI driver on a cluster running on vSphere, you must have the follo
 
 These components are still supported, but are deprecated; support for them will be removed in a future version of {product-title}.
 
+[id="ocp-4-10-installation-and-upgrade-alibaba-ipi"]
+==== Installing a cluster on Alibaba Cloud using installer-provisioned infrastructure
```

Comment:
add (Technology Preview) after the heading. An example from 4.9 Release Notes: https://docs.openshift.com/container-platform/4.9/release_notes/ocp-4-9-release-notes.html#ocp-4-9-sandboxed-containers-tp